text-generation-inference/docker/Dockerfile

```diff
@@ -74,7 +74,7 @@ WORKDIR /pyserver
 COPY text-generation-inference/server server
 COPY --from=tgi /tgi/proto proto
 RUN pip3 install -r server/build-requirements.txt
-RUN VERBOSE=1 BUILDDIR=/pyserver/build PROTODIR=/pyserver/proto VERSION=${VERSION} make -C server gen-server
+RUN VERBOSE=1 BUILDDIR=/pyserver/build PROTODIR=/pyserver/proto TGI_VERSION=${TGI_VERSION} VERSION=${VERSION} make -C server gen-server
```

Comment:
this is TGI's make, and it does not need the TGI_VERSION variable and it does not use it, so I would say this is not necessary.

Reply:
Actually this is the tpu server Makefile, and it does use the variable to fetch the .proto, but only if PROTODIR is not specified. So it is indeed useless here.